src/version.ts

```
@@ -0,0 +1,71 @@
+import {CommandsMap} from './command';
+import dirname from './dirname';
+import {versionNoRegisteredCommands, versionRegisteredCommands, versionCurrentVersion} from './text';
+import {join} from 'path';
+const pkgDir = require('pkg-dir');
+const packagePath = pkgDir.sync(dirname);
+
+interface ModuleVersion {
+	name: string;
+	version: string;
+	group: string;
+}
+
+interface PackageDetails {
+	name: string;
+	version: string;
+}
+
+export function readPackageDetails(packageDir: string): PackageDetails {
+	const data = require(join(packageDir, 'package.json'));
```

OK so I'm adding a check for `package.json`.. I plan on just using an empty object if the file isn't there so that it'll just use the defaults. Do you think I should `console.warn` about this, or just silently continue?